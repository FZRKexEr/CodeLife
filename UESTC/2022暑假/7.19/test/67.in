10
lodovzbtszfefgioyusraqywtxetivrlmontgcugmxjgmtbgrbrykpuilxfcklpqqmxzkjfqkuuokmggeuegtxpxtnrfrpjwtjqo
2
iybyuxgrviuvpkimjwkwtcobdhsymtxaplvsduooaderffdjcczjmjqlvxnkvthpqopotifxrdqzkqfakixcarwapbvshkuhydql
5
bbabrmzevcyerqnpymmiugismazhgrfjuvqvxfvqckwciixjejcbxjacybydptnmuhrzdifqkthspqdkwhfcddkrkedygxamsvta
4
olbdzpbnrbaqhddlebnrosnvjtezfpkvlvfrshqdnwzqrhxfuelxokuctwbfhtonxvapjdgmguyfterpxxdjdewclsxqdmstqtkb
8
ankjnogqlhxtiyjhqhlltqkaivajmluqrejvkbkhmlqoycaogdfzfnytimxmuywqreidfnefbdvbajhfzhdknceevqawawbucebf
10
mcjthdbguaqlpggpxzspjbxgbwaqbebkajozrpsmnmreqocljezaraewqhvukvobqgxkprbeqonsgvdwnmiipsbzcthqtiptrmix
6
dntlnvzndbiwayjfycpzkjjfjarqspbpasshmqobivlkwnoftgasxgeizadpvdzmbcjgsjpgefenifmrqnsnbsowfjhjqmbrldeg
2
ixkyfrfplpupzugnrlvgrskxvorvdlwbybkdendayzggjmvitkahhcolrjvaduopzhkljsgnpibckzboubmadopcryicptiuelgb
7
uwewlwpmeodwkxqwvikuwkiakbzxwnpnhlubulowyrhpqqqdtzqkviulubkjeysrycnupofvuzewowhzkujhkdnijjxsizkboejz
8
qlxgajvoitwtulytpdnhjhpelmlgvbdvhzkcpnnbplvrppcxogoezvaggqkkllbjmmbxqedkujckjrbclvslfbvvmdsbebkdeykb
2
