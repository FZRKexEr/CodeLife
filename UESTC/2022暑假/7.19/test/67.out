65980984
1048576
33554432
4096
1024
65536
65980984
16384
4096
65980984
